code/datums/uplink_item.dm

```diff
@@ -871,9 +863,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "A manual that teaches a single user tactical Close-Quarters Combat before self-destructing. Does not restrict weapon usage, but cannot be used alongside Gloves of the North Star."
 	reference = "CQC"
 	item = /obj/item/CQC_manual
-	gamemodes = list(/datum/game_mode/nuclear)
 	cost = 13
-	surplus = 0
 
```

Comment:
Recommend adding a `cant_discount = TRUE` and `refundable = TRUE` here, since martial arts are **!!STRONK!!** and accidental purchasing it would be a bit of a pain.

Reply:
fair enough

Reply:
I'd argue against that.... since we allow MUCH stronger things on discount than cqc.

Reply:
It's not about strength, but it's the same reason as carp, if you get it, you can't unget it, sniper rifle can oneshot a guy, but you can still disarm em and pick it up. Sure, chainsaw is nodrop, but you can just kill them and clone them.

Reply:
Does CQC persist through being killed and cloned?

Reply:
It does persist through cloning now. As does any martial art